### PR TITLE
Added the start of the uninstall wrapper function to twrl

### PR DESCRIPTION
**Pull request by caughtquick**
Pretty much all, will need to read through docs and need help to finish.

### CODE REVIEW

**Comment by FjorgeTheHuman** [comment]
I believe we have our own exception type for when packages are not found. I forget the name of it right now.

**Comment by caughtquick** [comment]
I'm not sure what that is, ill look through the docs but if someone knows it off the top of their head place tell me

**Comment by FjorgeTheHuman** [comment]
Lol what docs? Also this might sound like stupid python and that's because it is, but the `N` in `None` must be capital on line 86

**Comment by BBaoVanC** [comment]
> Lol what docs? Also this might sound like stupid python and that's because it is, but the `N` in `None` must be capital on line 86

This is why testing stuff helps.

File: twrl [hunk 1]
```diff
@@ -77,3 +77,14 @@ elif op == "refresh" or op == "sync":
 		print(repo)
 		handle.register_syncdb(repo,pyalpm.SIG_DATABASE)
 	print("Done!")
+elif op == "uninstall" or op == "remove"
+        try:
+            query = [sys.argv[2]]
+        except Exception: 
+            query = input("Package Name: ")
+        pkg db.get_pkg(query)
```
Comment (FjorgeTheHuman) [inline twrl:85]: Shouldn't this be `pkg = db.get_pkg(query)`?

**Review by FjorgeTheHuman — CHANGES_REQUESTED** [review]

File: twrl (hunk 1 above)
Comment (caughtquick) [reply twrl:85]: fixed it, will push it and edit the PR

File: twrl [hunk 2]
```diff
@@ -77,3 +77,14 @@ elif op == "refresh" or op == "sync":
 		print(repo)
 		handle.register_syncdb(repo,pyalpm.SIG_DATABASE)
 	print("Done!")
+elif op == "uninstall" or op == "remove"
```
Comment (FjorgeTheHuman) [inline twrl:80]: missing colon at EOL

**Review by FjorgeTheHuman — CHANGES_REQUESTED** [review]

File: twrl [hunk 3]
```diff
@@ -77,3 +77,14 @@ elif op == "refresh" or op == "sync":
 		print(repo)
 		handle.register_syncdb(repo,pyalpm.SIG_DATABASE)
 	print("Done!")
+elif op == "uninstall" or op == "remove":
+        try:
+            query = [sys.argv[2]]
```
Comment (FjorgeTheHuman) [inline twrl:82]: Unneeded brackets here

**Review by FjorgeTheHuman — CHANGES_REQUESTED** [review]

File: twrl (hunk 3 above)
Comment (FjorgeTheHuman) [reply twrl:82]: I'm really sorry for spamming all these reviews, I'm pretty new to git and github so I don't know how to use it that well.

Comment (FjorgeTheHuman) [reply twrl:82]: If you know how to condense all of these into one review please tell me.

Comment (caughtquick) [reply twrl:82]: I think you do what you can see all, then I push. Ill fix this now

Comment (FjorgeTheHuman) [reply twrl:82]: Ok, well this is the last issue I can see for now anyways

File: twrl [hunk 4]
```diff
@@ -77,3 +77,14 @@ elif op == "refresh" or op == "sync":
...
 		handle.register_syncdb(repo,pyalpm.SIG_DATABASE)
 	print("Done!")
+elif op == "uninstall" or op == "remove":
+        try:
+            query = sys.argv[2]
+        except Exception: 
+            query = input("Package Name: ")
+        pkg = db.get_pkg(query)
+        if pkg == none:
+            raise Exception(f"{query} does not exist.")
```
Comment (FjorgeTheHuman) [inline twrl:87]: `libtwirl` provides the `PackageNotFoundError` exception type which I think would be fitting here.

**Review by FjorgeTheHuman — APPROVED** [review]
Looks good, but see my comment on the exception raising.

File: twrl (hunk 4 above)
Comment (caughtquick) [reply twrl:87]: ill get that in now, thanks fjorge

Comment (thewarsawpakt) [reply twrl]: is this resolved?

**Review by mugman174 — APPROVED** [review]
:smile: works for me

**Review by thewarsawpakt — APPROVED** [review]
approved.

**Comment by mugman174** [comment]
Ok close this now

**Pull request merged**